packages/toggleInputList/components/ToggleInputList.tsx

```diff
@@ -140,7 +143,7 @@ const ToggleInputList = ({
       };
 
       return (
-        <li className={padding("top", "xs")} key={id}>
+        <li className={padding("top", "xs")} key={itemId}>
```

Comment:
Why did you need to set the key to `itemId`?
It is cast above in the destructured props so that id = itemId like so:
`id: itemId,`